services/harmony/app/util/cmr.ts

```diff
@@ -250,11 +251,19 @@ export interface CmrRelatedUrl {
 
 export interface CmrQuery
   extends NodeJS.Dict<string | string[] | number | number[] | boolean | boolean[] | null> {
+  bounding_box?: string;
```

Comment:
I think I made this change early on thinking we'd convert the WKT to CMR query parameters. Since we're instead using a GeoJSON shapefile, we might want to remove the circle, line, point, polygon fields - probably doesn't matter.

Reply:
done.